livefs_edit/actions.py

```diff
@@ -579,16 +598,19 @@ def unpack_initrd(ctxt, target='new/initrd'):
     ctxt.run(['unmkinitramfs', ctxt.p(f'new/iso/{initrd_path}'), lower])
     overlay = ctxt.add_overlay(lower, target)
 
-    if 'early' in os.listdir(target):
+    multipart_dirs = set(['early', 'cpio1'])
+    if bool(multipart_dirs & set(os.listdir(target))):
```

Comment:
Nice. I'm still curious what the directory is supposed to look like on the `_pre_repack_single` code path since we were trying to generalize this logic but this reads nice too!